README.md

```diff
@@ -1,5 +1,6 @@
 # ccd-case-document-am-client
 
+[![API v1](https://img.shields.io/badge/API%20Docs-v1-e140ad.svg)](https://hmcts.github.io/reform-api-docs/swagger.html?url=https://hmcts.github.io/reform-api-docs/specs/document-management-store-app.json)
```

Comment:
is this the correct documentation ?

`document-management-store-app`

Reply:
I had thought this had been updated in Sateesh's PR but I was wrong hence the incorrect copy + paste. Will update this once Sateesh generates a new Swagger link for Case Doc API.